### DISCUSSION TITLE
单个的“1”不能识别

### DISCUSSION THREAD

**Discussion by playgithub** [PaddlePaddle/PaddleOCR#13451] [Q&A]
如图，可能是太简单了，理解成直线了，怎么办？

![image](https://github.com/user-attachments/assets/263f99cb-a2d2-4e86-9527-c8b075d20027)

**Comment by UserWangZz** [comment]
det_db_box_thresh
det_db_unclip_ratio
use_dilation
det_db_score_mode
尝试调整上面参数看看是否可以获得更好的结果，如果不行，尝试微调模型。
https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_ch/inference_args.md
推理参数含义参考上面文档

**Comment by playgithub** [comment]
```
det_db_thresh = 0.1
det_db_box_thresh = 0.1
```
没用

```
use_dilation = True
```
也没用

看来得微调
